src/markdownPreview/edit.html

```diff
@@ -10,15 +17,30 @@
       </select>
     </p>
 
-    <p ng-if="vm.branches"><label for="branch">Branch</label>
+    <p ng-if="vm.branches">
+      <label for="branch">Branch</label>
+      <span class="grey glyphicon glyphicon-info-sign ng-scope" tabindex="0"
+            ng-focus="branchTooltipIsOpen = !branchTooltipIsOpen" tooltip-trigger="outsideClick"
+            uib-tooltip-html="branchTooltip" tooltip-trigger="'outsideClick'"
+            ng-mouseover="branchTooltipIsOpen = !branchTooltipIsOpen" tooltip-placement="top"
+            tooltip-is-open="branchTooltipIsOpen"
+            aria-label="A valid URL to your Jenkins server must be stored here"></span>
       <select name="branch" id="branch" class="form-control" ng-model="config.branch">
         <option ng-repeat="branch in vm.branches| orderBy: 'name'" value="{{branch.id}}">
           {{branch.name}}
         </option>
       </select>
     </p>
-    <label for="path">Path to Markdown File</label>
-    <input type="text" class="form-control" id="path" ng-model="config.path">
+    <p ng-if="vm.branches">
+      <label for="path">Path to Markdown File</label>
+      <span class="grey glyphicon glyphicon-info-sign ng-scope" tabindex="0"
+            ng-focus="pathToMarkdownTooltipIsOpen = !pathToMarkdownTooltipIsOpen" tooltip-trigger="outsideClick"
+            uib-tooltip-html="pathToMarkdownTooltip" tooltip-trigger="'outsideClick'"
+            ng-mouseover="pathToMarkdownTooltipIsOpen = !pathToMarkdownTooltipIsOpen"
+            tooltip-is-open="pathToMarkdownTooltipIsOpen"
+            tooltip-placement="top" aria-label="A valid path to a markdown file must be stored here"></span>
```

Comment:
Please refer to voice & tone when writing messages. “A valid path to a markdown file must be stored here” might be rewritten to “Enter the path to a markdown file”.
Add information if the format of the path has to follow a special format (no leading / for example)